pkg/eventcache/eventcache.go

```diff
@@ -131,21 +129,29 @@ func (ec *Cache) handleEvents() {
 		// process and lets copy it across.
 		if event.internal == nil {
 			event.internal, err = event.msg.RetryInternal(event.event, event.startTime)
+			if err != nil && event.color >= CacheStrikes {
+				if errors.Is(err, ErrFailedToGetParentInfo) {
+					errormetrics.ErrorTotalInc(errormetrics.EventCacheParentInfoFailed)
+				} else if errors.Is(err, ErrFailedToGetProcessInfo) {
+					errormetrics.ErrorTotalInc(errormetrics.EventCacheProcessInfoFailed)
+					eventcachemetrics.ProcessInfoError(notify.EventTypeString(event.event)).Inc()
+				}
+			}
```

Comment:
by looking at the code they seem to be monitoring 2 different things and would cause no harm and increase the observability of 2 things ? what do you say